src/packages/client/src/__tests__/integration/errors/union-validation/schema.prisma

```diff
@@ -6,7 +6,7 @@ generator client {
 
 datasource db {
   provider = "sqlite"
-  url      = env("DB_URL")
+  url      = "file:./dev.db"
```

Comment:
Nope, the 0B binary files are for updated tests and one is for the fixture used for development. I changed this because for some reason this test is flaky when using NAPI

Reply:
Maybe this is something we can look into today, could be related to the search paths I'm trying to fix.